networktest.go

```diff
@@ -243,8 +247,16 @@ func (l *lightningNode) Start(lndError chan error) error {
 		return err
 	}
 
+	// Wait until SSL certificate is created before using it
+	for !fileExists(l.cfg.SSLCert) {
+		time.Sleep(100 * time.Millisecond)
```

Comment:
this could ideally lock forever, right? have you thought of panicking after a max number of retries?